homeassistant/components/facebox/image_processing.py

```diff
@@ -66,7 +68,7 @@ def check_box_health(url, username, password):
     if username:
         kwargs["auth"] = requests.auth.HTTPBasicAuth(username, password)
     try:
-        response = requests.get(url, **kwargs)
+        response = requests.get(url, **kwargs, verify=False)
```

Comment:
Why add a feature to the use of HTTPS without verification? That eliminates 95% of the security benefit of using HTTPS. Why would a user want to use this option that provides a (mostly) false sense of security?